fastlane/README.md

```diff
@@ -34,27 +34,47 @@ xcode-select --install
 ```
 fastlane ios release_testflight
 ```
+Releases a new version to `TestFlight`. This lane uses the configuration mapped to `:testflight`.
+
+It has basically 3 main responsabilities: build number managing, app building, and deploy.
+
+- Reads the build number from `'CURRENT_PROJECT_VERSION'` xcode build configuration for the configuration in use.
+- Increments and saves this news value in the xcode build configuration.
+- Sets this incremented value in the `Info.plist` under the key `CFBundleVersion`, to use this bundle version for the app.
+- Builds the app using `gym` and `match` to get the signing identity. The provisioning profile in use is the one selected in xcode for the configuration mapped by `:testflight`.
+- Uploads the generated `.dsym` file to `Rollbar`.
+- Discards the value updated in `Info.plist`. Given this file is used for every configuration, these values are stored in xcode build configuration and just reflected in `Info.plist` when building.
+- Uploads the application to `TestFlight` using `pilot`.
+
+NOTE: It's important to note that in case the the version number needs to be updated, it has to be manually done, editing the `'CFBundleShortVersionString'` key from `Info.plist`.
```

Comment:
Sorry, one thing I don't understand.
In practice, each environment can have a different version, right? But the xcode build configuration is global right? Couldn't we in the Info.plist store the last version of each environment and take it from there, increment it and updated? Changing the `CFBundleVersion` only for building and reverting, but changing always these keys we add ?

Or this is simpler than I'm imagining?